blackbox/docs/config/session.rst

```diff
@@ -36,7 +36,10 @@ Supported Session Settings
 
 .. _conf-session-search-path:
 
-``search_path``: *schema names* (default: ``doc``)
+**search_path**
+  | *Default:* ``pg_catalog, doc``
+  | *Modifiable:* ``yes``
```

Comment:
please also update the beginning of the `Usage` section  to emphasise on the `modifiable` bit:

> To configure a **modifiable** session setting, use :ref:`SET <ref-set>` eg: